src/status_im/constants.cljs

```diff
@@ -217,7 +217,7 @@
 (def regx-community-universal-link #"((^https?://status.app/)|(^status-app://))c/([\x00-\x7F]+)$")
 (def regx-deep-link #"((^ethereum:.*)|(^status-app://[\x00-\x7F]+$))")
 (def regx-ens #"^(?=.{5,255}$)([a-zA-Z0-9-]+\.)*[a-zA-Z0-9-]+\.[a-zA-Z]{2,}$")
-(def regx-address #"^0x[a-fA-F0-9]{40}$")
+(def regx-address #".*0x[0-9a-fA-F]{40}$")
```

Comment:
Is this change to allow multichain addresses?

If that is the case, maybe we can rename this regx to be `regx-multichain-address`?

Reply:
Also, not an expert on Regular Expressions, but does the `*` allow any string before the address? Maybe we can be more explicit in the regex definition so we only allow alphanumeric strings separated by `:` like `eth:`, `opt:`, etc? I am sure ChatGPT can help with bulding that 😄

Reply:
Good suggestion I'll try that 😄

Reply:
@briansztamfater fixed!

Reply:
@OmarBasem I just realized this. Better to have two regexs, legacy and multichain. 
We should be able to detect both, we will need this in other parts of the app too: e.g https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?type=design&node-id=2937-659374&mode=design&t=Qew5GpIZJ0j07xPx-0

Reply:
@J-Son89 the regex I added can detect both legacy and multichain addresses